src/stylesheets/components/_autocomplete.scss

```diff
@@ -19,54 +32,67 @@
 
   /* Track */
   &::-webkit-scrollbar-track {
-    background: #f1f1f1;
+    @include u-bg('base-lighter');
   }
 
   /* Handle */
   &::-webkit-scrollbar-thumb {
-    background: #888;
+    @include u-bg('base');
     border-radius: 0;
   }
 
   /* Handle on hover */
   &::-webkit-scrollbar-thumb:hover {
-    background: #555;
+    @include u-bg('base-dark');
   }
-}
 
-.sds-autocomplete .sds-autocomplete__item {
-  border-top: 1px solid #ccc;
-  padding: 5px 10px;
-  cursor: pointer;
-}
+  /* Item */
+  .sds-autocomplete__item {
+    @include u-padding(1);
+    cursor: pointer;
+  }
 
-.sds-autocomplete .sds-autocomplete__item.sds-autocomplete__item--selected {
-  background-color: #0071bc;
-  color: white;
-}
+  /* Item hover or selected */
+  .sds-autocomplete__item:hover, .sds-autocomplete__item--selected {
+    @include u-bg('secondary');
+    @include u-text('white');
+  }
 
-// sds-autcomplete-selected
-.sds-autocomplete-selected .sds-autocomplete-selected__item {
-  border-radius: 2px;
-  background-color: color('primary-lighter');
-  border: 1px solid color('primary');
-  display: inline-block;
-  font-size: 14px;
-  margin-top: 5px;
-  margin-bottom: 0px;
-  padding: 3px 8px;
-  width: calc(100% - 16px);
-  max-width: 30rem;
-  position: relative;
-}
+  /* Item Group */
+  .sds-autocomplete__group {
+    div.sds-autocomplete__item {
+      @include u-bg('base-lightest');
+      @include u-text('semibold');
+    }
+
+    div.sds-autocomplete__item:hover, div.sds-autocomplete__item--selected {
+      @include u-bg('secondary');
+      @include u-text('white');
+    }
+
+    /* Child with group */
+    .sds-autocomplete__group {
+      .sds-autocomplete__item {
+        @include u-padding-left(2);
+      }
+
+      ul > li.sds-autocomplete__item {
+        @include u-padding-left(4);
+      }
+    }
+
+    /* Grandchildren Items */
+    & > ul > li.sds-autocomplete__item {
+      @include u-padding-left(2);
+    }
+  }
+
+  /* Disabled */
+  .sds-autocomplete__item--disabled, .sds-autocomplete__item--disabled:hover {
+    @include u-text('base');
+    @include u-bg('white');
+    @include u-text('italic');
+    cursor: not-allowed;
+  }
 
-.sds-autocomplete-selected .sds-autocomplete-selected__item--disabled,
-.sds-autocomplete[disabled='true'],
- input [disabled='disabled'] {
```

Comment:
If you don't use the "disabled" attribute the element can still receive focus which would be a problem for screen readers even though it will not trigger an automated 508 issue

Reply:
The component can still add disabled for 508 purposes, but it will also add the disabled class. I just chose to apply the styling against the class.